vscode emmet extension does not work anymore

```
Uncaught Error: Cannot find module 'vscode-emmet-helper'

Error: Cannot find module 'vscode-emmet-helper'
    at Function.Module._resolveFilename (/internal/modules/cjs/loader.js:582)
    at Function.Module._load (/internal/modules/cjs/loader.js:508)
    at Function.module._load (/workspace/theia/packages/plugin-ext-vscode/lib/node/plugin-vscode-init.js:98)
    at Module.require (/internal/modules/cjs/loader.js:637)
    at require (/internal/modules/cjs/helpers.js:22)
    at Object.<anonymous> (/tmp/vscode-unpacked/vscode-builtin-emmet-0.1.0.tgz/package/dist/extension.js:1)
    at n (/tmp/vscode-unpacked/vscode-builtin-emmet-0.1.0.tgz/package/dist/extension.js:1)
    at Object.t.getEmmetHelper (/tmp/vscode-unpacked/vscode-builtin-emmet-0.1.0.tgz/package/dist/extension.js:1)
    at t.DefaultCompletionItemProvider.provideCompletionItemsInternal (/tmp/vscode-unpacked/vscode-builtin-emmet-0.1.0.tgz/package/dist/extension.js:1)
    at t.DefaultCompletionItemProvider.provideCompletionItems (/tmp/vscode-unpacked/vscode-builtin-emmet-0.1.0.tgz/package/dist/extension.js:1)
    at /vs/editor/editor.main.js:12536
/vs/editor/editor.main.js:12536 Uncaught Error: a.updateExtensionsPath is not a function

TypeError: a.updateExtensionsPath is not a function
    at c (/tmp/vscode-unpacked/vscode-builtin-emmet-0.1.0.tgz/package/dist/extension.js:1)
    at Object.t.getEmmetHelper (/tmp/vscode-unpacked/vscode-builtin-emmet-0.1.0.tgz/package/dist/extension.js:1)
    at t.DefaultCompletionItemProvider.provideCompletionItemsInternal (/tmp/vscode-unpacked/vscode-builtin-emmet-0.1.0.tgz/package/dist/extension.js:1)
    at t.DefaultCompletionItemProvider.provideCompletionItems (/tmp/vscode-unpacked/vscode-builtin-emmet-0.1.0.tgz/package/dist/extension.js:1)
    at CompletionAdapter.provideCompletionItems (/workspace/theia/packages/plugin-ext/lib/plugin/languages/completion.js:36)
    at /workspace/theia/packages/plugin-ext/lib/plugin/languages.js:149
    at LanguagesExtImpl.withAdapter (/workspace/theia/packages/plugin-ext/lib/plugin/languages.js:121)
    at LanguagesExtImpl.$provideCompletionItems (/workspace/theia/packages/plugin-ext/lib/plugin/languages.js:149)
    at RPCProtocolImpl.doInvokeHandler (/workspace/theia/packages/plugin-ext/lib/api/rpc-protocol.js:207)
    at RPCProtocolImpl.invokeHandler (/workspace/theia/packages/plugin-ext/lib/api/rpc-protocol.js:192)
    at /vs/editor/editor.main.js:12536
```

In order to reproduce, install it from https://registry.npmjs.org/@theia/vscode-builtin-emmet/-/vscode-builtin-emmet-0.2.1.tgz

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,6 +6,7 @@
 - [security] Bump lodash.mergewith from 4.6.1 to 4.6.2
 - [plugin] Fixed `Converting circular structure to JSON` Error [#5661](https://github.com/theia-ide/theia/pull/5661)
 - [plugin] fixed auto detection of new languages [#5753](https://github.com/theia-ide/theia/issues/5753)
+- [vscode] unzip node_modules for built-in extensions [#5756](https://github.com/theia-ide/theia/pull/5756)
 
 Breaking changes:
 

--- a/packages/plugin-ext-vscode/src/node/plugin-vscode-file-handler.ts
+++ b/packages/plugin-ext-vscode/src/node/plugin-vscode-file-handler.ts
@@ -16,6 +16,7 @@
 
 import { PluginDeployerFileHandler, PluginDeployerEntry, PluginDeployerFileHandlerContext } from '@theia/plugin-ext';
 import { injectable } from 'inversify';
+import * as fs from 'fs-extra';
 import * as path from 'path';
 import { getTempDir } from '@theia/plugin-ext';
 
@@ -41,6 +42,13 @@ export class PluginVsCodeFileHandler implements PluginDeployerFileHandler {
 
         const unpackedPath = path.resolve(this.unpackedFolder, path.basename(context.pluginEntry().path()));
         await context.unzip(context.pluginEntry().path(), unpackedPath);
+        if (context.pluginEntry().path().endsWith('.tgz')) {
+            const extensionPath = path.join(unpackedPath, 'package');
+            const vscodeNodeModulesPath = path.join(extensionPath, 'vscode_node_modules.zip');
+            if (await fs.pathExists(vscodeNodeModulesPath)) {
+                await context.unzip(vscodeNodeModulesPath, path.join(extensionPath, 'node_modules'));
+            }
+        }
 
         context.pluginEntry().updatePath(unpackedPath);
         return Promise.resolve();

